exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>What does it mean to "pin the version"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>What about:</p><p>We usually recommend to install a specific the version, such that there are no surprises for which version gets installed.</p><p>?</p>

Reply:
<p>That sounds much more clear. </p>

Reply:
<p>Done</p>